src/tutorials/0005-regular-files-api/04.md

```diff
@@ -3,28 +3,35 @@
 
 The `cat` method first searches your own node for the file requested, and if it can't find it there, it will attempt to find it on the broader IPFS network. Because of the way cryptographic hashing works, the content you're searching for is guaranteed to match the file found, regardless of which peer is hosting it.
 
-Data returned by the `cat` method comes in the form of a `Buffer`. A `Buffer` is just a raw collection of bytes and as such doesn't make any assumptions about encodings or the type of data it contains. However, if we know the file being retrieved is a plain text file such as a `.txt`, we can convert its buffered contents to a UTF-8 string (an interpretation of those raw bytes) by calling the JavaScript method `.toString()`.
+The `cat` method returns an Async Iterable, that iterates over the file's chunks of data, i.e. Buffers.
+
+A `Buffer` is just a raw collection of bytes and as such doesn't make any assumptions about encodings or the type of data it contains. However, if we know the file being retrieved is a plain text file such as a `.txt`, we can convert its buffered contents to a UTF-8 string (an interpretation of those raw bytes) by calling the JavaScript method `.toString()`.
 
 So if you had the CID for a text file in an IPFS node, you could retrieve the file's contents as a readable string like so:
 
 ```javascript
-let bufferedContents = await ipfs.cat('QmWCscor6qWPdx53zEQmZvQvuWQYxx1ARRCXwYVE4s9wzJ') // returns a Buffer
-let stringContents = bufferedContents.toString() // returns a string
+const concat = require('it-concat')
+
+const bufferedContents = await concat(ipfs.cat('QmWCscor6qWPdx53zEQmZvQvuWQYxx1ARRCXwYVE4s9wzJ')) // returns a Buffer
+const stringContents = bufferedContents.toString() // returns a string
 ```
-Notice how the example above waits for the results of the `cat` method before converting the value to a string. To accomplish that in a single line of code, you'd need to wrap the `await` statement in parentheses like so:
+
+Notice how the example above waits for the concatenation of the results of the `cat` method (using the [`it-concat`](https://www.npmjs.com/package/it-concat) package) before converting the value to a string. To accomplish that in a single line of code, you'd need to wrap the `await` statement in parentheses like so:
 
 ```javascript
-let stringContents = (await ipfs.cat(cid)).toString() // returns a string
+const stringContents = (await concat(ipfs.cat(cid))).toString() // returns a string
 ```
```

Comment:
I don't think these single line versions add any value to the turorial. Saving a line of code should not be a concern for a newcomer (or arguably any developer).